Gruntfile.js

```diff
@@ -0,0 +1,81 @@
+/*
+ * flopmang
+ * https://github.com/dylancwood/flopmang
+ *
+ * Copyright (c) 2014 Dylan Wood
+ * Licensed under the MIT license.
+ */
+
+'use strict';
+
+module.exports = function (grunt) {
+
+    // Project configuration.
+    grunt.initConfig({
+        watch: {
+            dev: {
+                files: ['lib/*.js', 'test/*.js'],
+                tasks: ['jshint', 'test']
+            }
+        },
+
+        jshint: {
+            all: [
+                'Gruntfile.js',
+                'lib/*.js',
+                'test/**.js'
+            ],
+            options: {
+                jshintrc: '.jshintrc',
+            },
+        },
+
+        jscs: {
+            src: {
+                options: {
+                    config: '.jscs.json'
+                },
+                files: {
+                    src: [
+                        'Gruntfile.js',
+                        'lib/*.js',
+                        'test/**.js'
+                    ],
+                }
+            },
+        },
+
+        mochacli: {
+            options: {
+                files: 'test/*_test.js'
+            },
+            spec: {
+                options: {
+                    reporter: 'spec',
+                    timeout: 10000
+                }
+            }
+        },
+
+        bump: {
```

Comment:
Bump is a tool to ease implementation of semver. Since this is Ross' repo, it's really up to him how he decides to version it. IMO, COINS should use semver, and it would be nice if this repo did as well.